### DISCUSSION TITLE
GPU start/stop speeds

### DISCUSSION THREAD

**Discussion by perdyqueue23** [Rem0o/FanControl.Releases#2111] [General]
Hi, I have an aftermarket RTX 3080 for which I've set a custom behaviour - I want it to go from zero fan to active from 62 degrees, but once active, remain on until temps drop below 52.

I've set a mix curve as below, but after fans have ramped up at 62, they don't switch off again until exactly 40 degrees, which isn't a value I've set anywhere. The GPU default behaviour doesn't do that either. How can I remedy this?

![image](https://github.com/Rem0o/FanControl.Releases/assets/149584190/1ac603d3-bfa7-48b3-8b7c-1c17dc077886)

Thanks.

Edit: Tried LHM and NVAPI, both same.

Edit 2: Force apply didn't work

Edit 3: Read the description properly... please ignore this post

**Comment by Rem0o** [comment]
What's the GPU default behavior exactly? At what temperature does it go back to 0 rpm normally? Because you can't override that.

**Comment by perdyqueue23** [comment]
Yeah fair, and I've inspected the default behaviour - essentially the fan can stop anywhere from like 50 to 40 degrees depending on variables I haven't tested thoroughly. So it behaves the same actually, just the inconsistency was throwing me off. Thanks for replying.

**Reply by Rem0o** [reply]
I would adjust my trigger to replicate that behavior.  Make sure you got the 0% override "on" also.
